packages/psss/src/routes/PrivateRoute.js

```diff
@@ -33,14 +34,7 @@ export const PrivateRouteComponent = ({
       if (authCheck) {
         const isAuthorised = authCheck(currentUser, match);
         if (!isAuthorised) {
-          return (
-            <Redirect
-              to={{
-                pathname: redirectTo,
-                state: { from: location },
-              }}
-            />
-          );
+          return <UnauthorisedView />;
```

Comment:
Rather than redirect, keep the url and show an unauthorised view as it is more informative for the user to see what the url they typed in was.